source/conf.py

```
@@ -8,37 +8,47 @@
 
 # -- General configuration ---------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
 
 import os, sys
-sys.path.insert(0, os.path.abspath('../build'))  # if you’ll import your python package
 
-extensions = ["breathe", 
-              "sphinx_immaterial",
-             # "sphinx.ext.autodoc"
-              ]
+# sys.path.insert(0, os.path.abspath("../repos/pygenogrove"))  # if you’ll import your python package
```

Ruff (0.13.3)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)